openadmet/models/architecture/mtenn.py

```diff
@@ -0,0 +1,86 @@
+from lightning import pytorch as pl
+import torch
+from typing import ClassVar
+from loguru import logger
+from mtenn.config import SchNetModelConfig
+from openadmet.models.architecture.model_base import TorchModelBase
+from openadmet.models.architecture.model_base import models as model_registry
+
+
+class MTENNLightningWrapper(pl.LightningModule):
+    def __init__(self,
+                 model_config: SchNetModelConfig,
+                 loss_fn = torch.nn.MSELoss(),
+                 lr = 1e-4):
+         super().__init__()
+         self.model = model_config.build()
+         self.loss_fn = loss_fn
+         self.lr = lr
+
+    def training_step(self, batch, batch_idx):
+        data_batch, target_batch = batch
+        batch_loss = 0
+
+        for data, target in zip(data_batch, target_batch):
+            for k, v in data.items():
+                data[k] = v.to(self.device)
+            pred,_ = self.model(data)
+            loss = self.loss_fn(pred, target.unsqueeze(0).to(self.device))
+            batch_loss += loss
+
+        avg_loss = batch_loss / len(data_batch)
+        self.log('train_loss', avg_loss)
+        return avg_loss
+
+    def configure_optimizers(self):
+        return torch.optim.AdamW(self.model.parameters(), lr=self.lr)
+
+
+
+@model_registry.register("MTENNSchNetModel")
+class MTENNSchNetModel(TorchModelBase):
+    """
+    """
+
+    type:ClassVar[str] = "MTENNSchNetModel"
+    #metric_list : list = ['mae']
+    model_params: dict = {}
+
+    #from params?
+    # train here?
+
+    def build(self, scaler=None): #what is scaler?
+        """
+        Prepare the model
+        """
+        if not self.estimator:
+           #i think scaler is chemprop specific, but if not i will add loop back in
+            model_config = SchNetModelConfig(**self.model_params)
+            self.estimator = MTENNLightningWrapper(model_config)
+        else:
+            logger.warning("Model already exists, skipping build.")
+
+    def from_params(self, params):
```

Comment:
Should be easy to implement this, I am also thinking this method might be superfluous in the end, will raise issue.

Reply:
We can just leave this blank for now @khuddzu, see #129. Abstract base class will require it to be implemented but just leave it as `pass`